src/Security.tsx

```diff
@@ -48,7 +48,8 @@ const Security: React.FC<{
     }
 
     // Add default restoreOriginalUri callback
-    if (oktaAuth.options.restoreOriginalUri && restoreOriginalUri) {
+    // props.restoreOriginalUri is required, therefore if options.restoreOriginalUri exists, there are 2 callbacks
+    if (oktaAuth.options.restoreOriginalUri) {
```

Comment:
addresses TS error, `restoreOriginalUri` is always defined therefore always evaluates to `true`